### DISCUSSION TITLE
Chatbot which uses the whisper API

### DISCUSSION THREAD

**Discussion by hoani** [openai/whisper#1294] [Show and tell]
Hey team,

I put a Raspberry Pi inside a speaker, which uses whisper and chatGPT to build out a chatbot that you can have conversations with without typing.

![P_20230416_191127](https://user-images.githubusercontent.com/4222666/235093381-abbc3488-6a63-4d5d-8aa7-69036fd01c7e.jpg)

The build video is here:

https://youtu.be/rzS5zLpd1os

It is mostly written in Go, and the code is here https://github.com/hoani/chatbox

**Comment by candideu** [comment]
Nice!!

**Comment by TheAlphamerc** [comment]
That's amazing🎉
